src/atom.coffee

```diff
@@ -572,6 +581,13 @@ class Atom extends Model
     dimensions = @getWindowDimensions()
     @state.windowDimensions = dimensions if @isValidDimensions(dimensions)
 
+  storeWindowBackground: ->
```

Comment:
@mnquintana I'm not sure it will, I think that was an underlying Chrome issue that is still present.